server/controllers/reportsRouter/getReports.ts

```diff
@@ -0,0 +1,17 @@
+import { Request, Response, NextFunction } from 'express';
+import { Contact } from '../../database/models';
+
+const getReports = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const reports = await Contact.findAll({
+      attributes: {
+        exclude: ['createdAt', 'updatedAt'],
+      },
+    });
+    res.json({ message: 'Success', data: { reports } });
```

Comment:
You need to send the reports depending on the page.
And rename the Contact model to Report so we have the same name in all files.